src/gpu/locks.rs

```diff
@@ -50,7 +77,7 @@ impl Drop for GPULock {
 pub(crate) struct PriorityLock(File);
 impl PriorityLock {
     pub fn lock() -> PriorityLock {
-        let priority_lock_file = tmp_path(PRIORITY_LOCK_NAME);
+        let priority_lock_file = tmp_path(PRIORITY_LOCK_NAME, UniqueId::try_from(PRIORITY_LOCK_UID).unwrap());
```

Comment:
I propose making the `tmp_path()` taking an option for the UUID, so you would just pass in `None` here instead having a "fake" UUID without much benefits.